### ISSUE TITLE
Please provide the AppImage file for download

### ISSUE THREAD

**Issue by probonopd** [aforensics/HiddenVM#10]
Please provide the AppImage file for download, not inside a zip or other file.

The reasons are explained at 
https://github.com/AppImage/AppImageKit/wiki/Creating-AppImages/cc2441518975caca23e9ce2dba6f08a22c678d1e#common-mistake

Thank you very much.

**Comment by aforensics** [comment]
Thanks for the feedback. The main reason we archive it is to preserve x permission. We can certainly publish just the appimage file for users that prefer setting permission instead of unpacking an archive. The reasons listed in that link (aside from "the beauty of the AppImage format") don't seem applicable to how HiddenVM is used and distributed (appimaged, central catalog).

**Comment by probonopd** [comment]
> We can certainly publish just the appimage file for users that prefer setting permission instead of unpacking an archive.

Yes, that would be great. (If you like, you can publish it alongside the existing zip.)

**Comment by animik** [comment]
@probonopd It seems we're don't apply to the context for the reason you cite to provide our AppImage outside a ZIP. HVM and Tails is a very different way of computing. We think we're the first 'app for Tails' that's been created, or we're certainly quite rare in that regard at this point.

To me, important to make the download experience as easy and non-confusing for the beginner as possible. For now our 'Releases' page is the only way people get it. It could look confusing to have both a ZIP and a raw AppImage alongside each other. Are they different versions, which one do I download - both? To make less confusing, we'd have to label them in the file names and explain things further, and it'd upset the sha512 file too. I love providing options, but in this case the cons outweigh the pros.

@aforensics what do you think? Just keep it the way it is because our distribution method is different to the standard persistent host OS AppImage model? And we wait to see if AppImage can be distributed as executable before we change it?

**Comment by aforensics** [comment]
Yeah IMO this is a non-issue. Publishing both a zip and an appimage would result in users downloading the appimage file and trying to execute it by double clicking, and that wouldn't work. Thanks @probonopd, but I don't think we will do this.

**Comment by probonopd** [comment]
> some users downloading the appimage file and trying to execute it by double clicking, and that wouldn't work. Thanks @probonopd, but I don't think we will do this.

Looks like I need to learn more about how HVM and Tails work. So far, I literally download every AppImage from the download page, make it executable, and double-click it. After all, that is what the AppImage format has been designed for. But I take it that HVM and Tails apparently can't do this?
